tests/contracts/error/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "error"
+version = "0.1.0"
+authors = ["Kristoffer Ström <kristoffer@dusk.network>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+dusk-abi = { git = "https://github.com/dusk-network/dusk-abi", default-features = false }
+
+[profile.release]
+opt-level = 'z'
```

Comment:
I noticed that opt-level = 'z' doesn't produce in our case smaller binary size, that's why I removed them from a previous patch (they're actually slightly bigger using it).